app/js/displayCategories.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Compares 2 values and allows us to check a conditional statement in HBS
+ *
+ * @param arg1 first item to compare, the id of the current category from the API
+ * @param arg2 second item to compare against first item, the value of 1 which is the id that doesnt need a Delete button
+ * @param opts - option to run the code inside result of 'IF' statement on  line 5 categoryItems.hbs
+ */
+Handlebars.registerHelper('ifNotEquals', function(arg1, arg2, options) {
```

Comment:
this feels like it doesnt belong in this file. id suggest putting it in something like `utils.js` or `populateHandlebars.ts`